README.md

```diff
@@ -28,53 +28,84 @@ There are three different modes for the Moodle user to link files from oCIS to M
 
 ## Installation
-
-1. Install [ocis](https://doc.owncloud.com/ocis/next/quickguide/quickguide.html)
-   - NOTE: if you want to run ocis on `localhost` you need to create TLS certificates, make your system trust them and start ocis with those certificates. e.g. on Debian based systems:
-     ```bash
-     openssl req -x509  -newkey rsa:2048 -keyout ocis.pem -out ocis.crt -nodes -days 365 -subj '/CN=localhost'
-     sudo cp ocis.crt /usr/local/share/ca-certificates
-     sudo update-ca-certificates
-     OCIS_INSECURE=true \
-     PROXY_HTTP_ADDR=0.0.0.0:9200 \
-     OCIS_URL=https://localhost:9200 \
-     PROXY_TRANSPORT_TLS_KEY=./ocis.pem \
-     PROXY_TRANSPORT_TLS_CERT=./ocis.crt \
-     ./ocis server
-     ```
-     :exclamation: Having set `OCIS_INSECURE=true` is not recommended for production use! :exclamation:
-2. Install moodle and this plugin
-   - Development environment with docker
-     ```bash
-     git clone https://github.com/moodle/moodle.git --branch MOODLE_402_STABLE --single-branch --depth=1
-     cd moodle/repository/
-     git clone https://github.com/owncloud/moodle-repository_ocis.git ocis
-     cd ocis
-     composer install
-     cd ../../../
-     git clone https://github.com/moodlehq/moodle-docker.git
-     cd moodle-docker
-     export MOODLE_DOCKER_WWWROOT=<path-of-your-moodle-source-code>
-     export MOODLE_DOCKER_DB=pgsql
-     export MOODLE_DOCKER_PHP_VERSION=8.1
-     cp config.docker-template.php $MOODLE_DOCKER_WWWROOT/config.php
-     bin/moodle-docker-compose up -d
-     bin/moodle-docker-wait-for-db
-     bin/moodle-docker-compose exec webserver php admin/cli/install_database.php --agree-license --fullname="Docker moodle" --shortname="docker_moodle" --summary="Docker moodle site" --adminpass="admin" --adminemail="admin@example.com"
-     ```
-   - Other installations
-     - [Install and run moodle](https://docs.moodle.org/402/en/Installing_Moodle)
-     - copy / clone the code of the repository into the `repository/ocis` folder of your moodle installation
-     - run `composer install` inside of the `repository/ocis` folder
+1. Install moodle and this plugin
+    - Development environment with docker:
+      ```bash
+      # get moodle from git
+      git clone https://github.com/moodle/moodle.git --branch MOODLE_402_STABLE --single-branch --depth=1
+      # get and install this plugin including it's dependencies
+      cd moodle/repository/
+      git clone https://github.com/owncloud/moodle-repository_ocis.git ocis
+      cd ocis
+      composer install
+      # get docker containers for moodle developers
+      cd ../../../
+      git clone https://github.com/moodlehq/moodle-docker.git
+      cd moodle-docker
+      # some general settings for moodle
+      export MOODLE_DOCKER_WWWROOT=<path-of-your-moodle-source-code>
+      export MOODLE_DOCKER_DB=pgsql
+      export MOODLE_DOCKER_PHP_VERSION=8.1
+      cp config.docker-template.php $MOODLE_DOCKER_WWWROOT/config.php
```

Comment:
`sudo cp config.docker-template.php $MOODLE_DOCKER_WWWROOT/config.php`

Reply:
you should not need `sudo` here, because the `$MOODLE_DOCKER_WWWROOT` should be owned by your user